Profiler_Test/ProfilerTest.cs

```diff
@@ -8,82 +8,81 @@
 
 namespace Cqse.Teamscale.Profiler.Dotnet
 {

```

Comment:
Please undo formatting changes, as this makes merging hell. Afaik we should use tabs anyways...

Reply:
reverted